public/scripts/ssooverride.js

```diff
@@ -83,33 +89,53 @@
         }, false);
       });
 
-      var thimbleAuth = new WebmakerLogin({
-        csrfToken: csrf,
-        showCTA: false
-      });
+      var basicQuery = "";
+
+      basicQuery += "?client_id=" + oauthClientId;
```

Comment:
I prefer your second approach, but I can't think of an easy elegant way to clone the default query object. I'll go with the array stuff I think.